WordPress/src/main/res/menu/edit_comment.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <menu xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+      xmlns:tools="http://schemas.android.com/tools">
     <item
         android:id="@+id/menu_save_comment"
         android:title="@string/save"
-        app:showAsAction="always" />
+        app:showAsAction="always"
+        tools:ignore="AppCompatResource"/>
```

Comment:
I'm a bit confused by this change - it seems the warning says we should use "app" namespace instead of "android" when we are using AppCompat. Why would we want to suppress the warning when we already use the correct namespace? What am I missing?

Reply:
Nothing, I don't know why this is changed either. Wondering if it's something auto-generated by IntelliJ or something - I don't feel like it's something I'd change. Undone anyway.